pkg/apis/ocs/v1alpha1/storagecluster_types.go

```diff
@@ -62,3 +62,22 @@ type StorageClusterList struct {
 func init() {
 	SchemeBuilder.Register(&StorageCluster{}, &StorageClusterList{})
 }
+
+// ToStorageClassDeviceSet converts a StorageDeviceSet object to a Rook
+// StorageClassDeviceSet object
+func (ds *StorageDeviceSet) ToStorageClassDeviceSet() rookalpha.StorageClassDeviceSet {
+	return rookalpha.StorageClassDeviceSet{
+		Name:                 ds.Name,
+		Count:                ds.Count,
+		Resources:            ds.Resources,
+		Placement:            ds.Placement,
+		Config:               ds.Config.ToMap(),
+		VolumeClaimTemplates: []corev1.PersistentVolumeClaim{ds.DataPVCTemplate},
+	}
+}
+
+// ToMap converts a StorageDeviceSetConfig object to a map[string]string that
+// can be set in a Rook StorageClassDeviceSet object
+func (c *StorageDeviceSetConfig) ToMap() map[string]string {
+	return nil
```

Comment:
The comment claims that the `ToMap()` function does some conversion, but in reality it always returns `nil`. This is OK right now, since we don't have anything as config input yet, but we should either mention that this function is not implemented yet or else implement it so that it will work when we add any config items as input.

Reply:
I'll add a comment explaining the `return nil`.